docs/how-to-contribute.md

```diff
@@ -49,10 +50,10 @@ To create a local kind cluster, run the `make kind-cluster` command.
 
 #### TL;DR
 ```
-kind create cluster
```

Comment:
Shouldn't this be replaced with `make development-start`?

Reply:
how-to-contribute.md gives instruction on how to build a permission-manager image and deploy it to kubernetes cluster.
 It is different from the local development one